Sources/Gui/Main.cpp

```diff
@@ -57,13 +57,23 @@
 #endif
 
 
+#ifdef _MSC_VER>=1900	// Visual Studio 2015 or higher
+extern "C" { FILE __iob_func[3] = { *stdin,*stdout,*stderr }; }
```

Comment:
Am I doing something wrong? Maybe I need to recompile libpng as well for VS2015?
Here's the error: http://puu.sh/noxbo/85f8401b92.png